### ISSUE TITLE
Accessibility issues caused by components

### ISSUE THREAD

**Issue by haggishunt56** [HMPO/hmpo-components#133]
The hmpo-forms and hmpo-select components are returning html validation errors which, in turn, are causing projects using these components to fail accessibility tests.

Steps to reproduce:
1. In a project implementing hmpo-form, the source of the compiled element will look something like this: `<form action="/some-action" method="POST" autocomplete="off" novalidate="true" spellcheck="false">`
Inputting this into the [W3C HTML validator](https://validator.w3.org/nu/#textarea) returns the error 
> Bad value true for attribute novalidate on element [form](https://html.spec.whatwg.org/multipage/#the-form-element).
The compiled element should be: `<form action="/some-action" method="POST" autocomplete="off" novalidate spellcheck="false">`

2. In a project implementing hmpo-select, the compiled element will include <option> tags with no label attribute. The label is included between the opening and closing tags, i.e. <option value="1_DMT">DMT</option>. This also causes an error on the validator as it should instead be <option value="1_DMT" label="DMT"></option>

**Comment by aliuk2012** [comment]
hmpo-form error is addressed as part of https://github.com/HMPO/hmpo-components/pull/199  commit 
https://github.com/HMPO/hmpo-components/pull/199/changes/f601968c36c1c453fda84d5d4607d739973c42ad
